bin/clang/memoizeVariableCaptureAnalyzer.cc

```diff
@@ -43,7 +43,7 @@ Questions? Contact sst-macro-help@sandia.gov
 */
 
 #include "memoizeVariableCaptureAnalyzer.h"
-#include "clang/Analysis/Analyses/ExprMutationAnalyzer.h"
+//#include "clang/Analysis/Analyses/ExprMutationAnalyzer.h"
```

Comment:
It's fine to disable this for now, but I am probably going to enable it soon.  It doesn't belong in master, but I don't see a point in disabling it at the moment.

Reply:
This does not exist in Clang < 8. It will take some work to selectively enable this. Removed for now.